benchmark/displace_mpi.py

```diff
@@ -0,0 +1,61 @@
+"""
+MPI implementation of displacement generation benchmark
+
+Created on Tue Jun 09 12:03:31 2020
+
+@author: Henry Liu
+"""
+
```

Comment:
Please add some more docs about this script-- I am not familiar with MPI detail.